library/src/main/java/kaaes/spotify/webapi/android/models/CategoriesPager.java

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Looks like we had it before, since I didn't add it :) 
No doubts, I will clean it out.